<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilesInfoSets Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="TOOLS_FILES_RUNORDER">
         <property name="title">
          <string>Tools/Files Run Order</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="RB_FILE_PER_FILE">
              <property name="text">
               <string>File per File</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="RB_TOOL_PER_TOOL">
              <property name="text">
               <string>Tool per Tool</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="TOOLS_RUN_ORDER">
         <property name="title">
          <string>Tools Run Order</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <widget class="QLabel" name="TOOL_PYMETRICS">
              <property name="text">
               <string>PyMetrics:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="TOOL_PYMETRICS_RUN_ORDER">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="TOOL_PYLINT">
              <property name="text">
               <string>PyLint:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="TOOL_PYFLAKES_RUN_ORDER">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="TOOL_PYFLAKES">
              <property name="text">
               <string>PyFlakes:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="TOOL_PYLINT_RUN_ORDER">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="SELECTED_FILEINFOSET_ACTIONS">
         <property name="title">
          <string>Selected FileInfoSet Actions</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="INFOSET_RENAME">
              <property name="text">
               <string>Rename</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="INFOSET_SELECT_ALL">
                <property name="text">
                 <string>Select All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="INFOSET_UNSELECT_ALL">
                <property name="text">
                 <string>Unselect All</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="INFOSET_TOGGLE_SELECTIONS">
              <property name="text">
               <string>Toggle Selections</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="INFOSET_SORT_ITEMS">
              <property name="text">
               <string>Sort Items</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="INFOSET_REMOVE_UNSELECTED_ITEMS">
              <property name="text">
               <string>Remove Unselected Items</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="INFOSET_DELETE">
              <property name="text">
               <string>Delete FileInfoSet</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSplitter" name="MAIN_SPLITTER">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTabWidget" name="NOTEBOOK_INFOSETS">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="INFOSET_1">
           <attribute name="title">
            <string>IFS_1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="INFOSET_2">
           <attribute name="title">
            <string>IFS_2</string>
           </attribute>
          </widget>
         </widget>
         <widget class="QTreeView" name="TREE_FILESYSTEM"/>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="INFOSET_NEW">
         <property name="text">
          <string>New FileInfoSet</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DIALOG_OK_CANCEL">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>DIALOG_OK_CANCEL</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DIALOG_OK_CANCEL</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
